docs/data/material/getting-started/templates/dashboard/components/OptionsMenu.tsx

```diff
@@ -29,25 +39,26 @@ export default function OptionsMenu() {
         onClick={handleClose}
         transformOrigin={{ horizontal: 'right', vertical: 'top' }}
         anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
+        sx={{
+          '& .MuiPaper-root': {
```

Comment:
Instead of hardcoding the class `.MuiPaper-root`, we should always try to import the classes object and use that. In this case it should be

```js
import { paperClasses } from '@mui/material/Paper';

// ...

    sx={{
        `& .${paperClasses.root}`: {

// ...
```

Same for `MuiListItemIcon-root` below.

This avoids typos (e.g., typing `MuiPaper-rot`), and it's easier to codemod for future changes.